front_end/panels/ai_chat/mcp/MCPMetaTools.ts

```
@@ -0,0 +1,204 @@
+              required: ['selected']
+            }
+          }
+        }]
+      });
+
+      if (!response.functionCall) {
+        throw new Error('No function call in LLM response');
+      }
+
+      const functionCall = response.functionCall;
+      const selectedKeys = (functionCall.arguments as any)?.selected || [];
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)